coast/TIDEGAUGE.py

```diff
@@ -738,17 +725,17 @@ def read_bodc_to_xarray(cls, fn_bodc, date_start=None, date_end=None):
         -------
         xarray.Dataset object.
         """
-        debug(f'Reading "{fn_bodc}" as a BODC file with {get_slug(cls)}')  # TODO Maybe include start/end dates
+        debug(f"Reading \"{fn_bodc}\" as a BODC file with {get_slug(cls)}")  # TODO Maybe include start/end dates
         try:
             header_dict = cls.read_bodc_header(fn_bodc)
             dataset = cls.read_bodc_data(fn_bodc, date_start, date_end)
         except:
```

Comment:
We should avoid using bare excepts. These will catch **everything**. Including system exits.